js/views/dataset.vue

```diff
@@ -169,6 +182,22 @@ export default {
         edit() {
             this.$go({name: 'dataset-edit', params: {oid: this.dataset.id}});
         },
+        archive() {
+        this.dataset.archived = new Date().toISOString();
```

Comment:
Somehow the `archive` quickly shows but then disappears in the second (probably due to a watch effect)?

Reply:
Couldn't reproduce... I lack a bit experience on this project JS part, any idea on why and how to fix that?

Reply:
I've been debugging this issue but couldn't figure it out yet for now.
Notes:
- it seems the watcher is successfully triggered on any update on dataset.archived
- it seems the function `addOrRemoveBadge`works as expected as it add the "archived" badge to the badges list and removes it as expected
Could it be some display logic error elsewhere in a badges component?

Reply:
BTW logic to add a "archived" badge on reuse page as well as been added to the PR.

Reply:
Finally understood the issue! It came from the return value beeing not formatted exactly as ISO and thus triggering the watch effect again. Fixed it by checking if the value is None or not when removing the badge : https://github.com/opendatateam/udata/pull/3104/commits/6107ab52c9f13318597818db6c3d75563f923b1c.

Reply:
@maudetes Amazing, well done! Shall I merge from main and merge?